shared/src/CommonClientResponseCodes.cs

```diff
@@ -10,21 +11,25 @@ namespace Microsoft.Azure.Devices.Shared
     /// <remarks>
     /// These status codes are based on the HTTP status codes listed here <see href="http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml"/>
     /// </remarks>
-    [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable", Justification = "To allow customers to extend this class we need to not mark it static.")]
-    public class StatusCodes
+    [SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable",
+        Justification = "To allow customers to extend this class we need to not mark it static.")]
+    public class CommonClientResponseCodes
     {
         /// <summary>
         /// Status code 200.
         /// </summary>
         public static int OK => 200;
+
         /// <summary>
         /// Status code 202.
         /// </summary>
         public static int Accepted => 202;
+
         /// <summary>
         /// Status code 400.
```

Comment:
Maybe add more details to the doc comments? Like 400 means client error, 202 means request has been accepted for processing but the processing has not been completed, 200 means request has succeeded?

Reply:
It doesn't need to necessarily need to mean that though. These code aren't being referenced directly within the sdk to represent a certain state, rather they are meant to be used by the users to represent the response of a command or a property update request. Their application logic would dictate what these codes mean for them => 202 for them could mean "this request completed successfully". It would not be 100% as per API specs, but they are free to make any choice they want.

How about we say: "As per IANA (Internet Assigned Numbers Authority) this code represents ..." instead, to help nudge them in the right direction?